packages/web-components/src/label/label.template.ts

```diff
@@ -0,0 +1,13 @@
+import { html } from '@microsoft/fast-element';
+import { Label } from './label.js';
+
+/**
+ * The template for the Fluent label web-component.
+ * @public
+ */
+export const template = html`
+  <template>
+    <slot></slot>
+    <span part="asterisk" class="asterisk" ?hidden="${(attr: Label) => !attr.required}">*</span>
```

Comment:
I'm curious about this:

```ts
?hidden="${(attr: Label) => !attr.required}"
```

This feels like it's copied from somewhere. Our pattern has been `?hidden="${x => !x.required}"`.

Reply:
yeah sorry just a habit as that's how we were doing it in our horizons repo.